Tasks/Dubinin/Task3/Task3.py

```diff
@@ -0,0 +1,56 @@
+import math
+
+while True:
+       L = input('Please enter maximum number symbol in the string (more than 35):')
+       if not L.isdigit():
+           print ('Incorrect input, please enter number')
+           continue
+
+       L = int(L)
+
+       if L<36:
+           print ('Incorrect input, please enter number more than 35')
+           continue
+       break
+
+
+with open("ftext.txt", 'w'): pass   #clean file
```

Comment:
I assume, you need a file to append some data later, but the 'a' mode can create a new file as well